hphp/hack/test/quickfixes/mixed_annotation_on_return_void_function.php.exp

```diff
@@ -0,0 +1,2 @@
+function foo(): void {}
+async function bar(): Awaitable<void> {}
```

Comment:
The test runner also prints the name of the quickfix, so we can check it's sensible. You'll need to do the same here.

I've put up a diff so that `make test` will run this and make it easier for you to spot in future pull requests.

```suggestion
Change to `void`
Change to `Awaitable<void>`


function foo(): void {}
async function bar(): Awaitable<void> {}
```